lib/node_modules/@stdlib/math/base/special/rcbrtf/docs/types/index.d.ts

```diff
@@ -0,0 +1,56 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/**
+* Computes the reciprocal cube root of a single-precision floating-point number.
+*
+* @param x - input value
+* @returns reciprocal cube root of `x`
+*
+* @example
+* var v = rcbrtf( 8.0 );
+* // returns 0.5
+*
+* @example
+* var v = rcbrtf( 1000.0 );
+* // returns ~0.1
+*
+* @example
+* var v = rcbrtf( 0.0 );
+* // returns Infinity
+*
+* @example
+* var v = rcbrtf( Infinity );
+* // returns 0.0
+*
+* @example
+* var v = rcbrtf( -8.0 );
+* // returns -0.5
```

Comment:
We have a wrong return value [here](https://github.com/stdlib-js/stdlib/blob/95eb45b833c5599f97ce22e56f12129fbd3b7ab7/lib/node_modules/%40stdlib/math/base/special/rcbrt/docs/types/index.d.ts#L45) in our `rcbrt`. I will fix that in a follow up PR.